pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[build-system]
+requires = ["setuptools>=67.7.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ghga_event_schemas"
+version = "0.13.6"
+description = "GHGA Event Schemas: A package that collects schemas used for events exchanged between GHGA service."
+readme = "README.md"
+authors = [
+    { name = "German Human Genome Phenome Archive (GHGA)", email = "contact@ghga.de" },
+]
+requires-python = ">=3.9"
+classifiers = [
+    "Development Status :: 1 - Planning",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "License :: OSI Approved :: Apache Software License",
+    "Topic :: Internet :: WWW/HTTP :: HTTP Servers",
+    "Topic :: Software Development :: Libraries",
+    "Intended Audience :: Developers",
+]
+dependencies = [
+    "jsonschema>=4.17.3,<5.0.0",
+    "pydantic[email]>=1.0.0,<2.0.0",
+    "dnspython<2.4.0",
```

Comment:
Did you mean:
```suggestion
    "dnspython>2.4.0,<3",
```

Reply:
I opened it up to >=2, <3

Reply:
Why is this dependency needed at all?

Reply:
I talked with Thomas and he said it was a relic of when there was a conflict with some transitive dependency - it can be removed. Seems it's only needed by the email validator that Pydantic uses.

Reply:
And that might have changed in Pydantic v2 anyway.